_sass/_base.scss

```diff
@@ -364,33 +364,34 @@ footer {
     }
   }
 
+//Footer Links
   .small {
     margin-top: 10px;
     ul {
       display: flex;
       list-style: none;
-      margin: 0;
-
+      margin-left: 0;
       li {
         margin-right: 10px;
       }
     }
   }
+//Footer Icons
```

Comment:
should we have an empty line here? just to keep it consistent with the rest of the file.

Reply:
@franciscoovazevedo Thanks for the feedback, will clean it up.

Reply:
@franciscoovazevedo done.